### DISCUSSION TITLE
Does --dup-check work with Import?

### DISCUSSION THREAD

**Discussion by cbgrey** [RhetTbull/osxphotos#801] [Q&A] [answered]
I'm running the "import" command on a folder that will accumulate photos over time. Thus, when it runs...it will have duplicates from what was imported on a previous run. I've tried using the --dup-check option but it doesn't seem to change behavior. 

Apple Photos will give me a warning the a file is a duplicate but does so on every file. Even if I check "Apply to All Duplicates" it still prompts on each file. 

Should -D --dup-check be preventing this? 

This is the command I'm running: 
`osxphotos import ./*.jpg --dup-check -V -T --album "{created.year}/{created.mm}" --split-folder "/"`
(I've also tried -D)

This is the dialog I get in Apple Photos: 
<img width="418" alt="image" src="https://user-images.githubusercontent.com/718238/193962221-1f1130e0-12a4-4066-87fb-93a6fd68543d.png">

**Comment by RhetTbull** [comment, marked as answer]
Hi. Sorry, apparently the help text on `--dup-check` isn't clear. In your case it's doing exactly what I intended `--dup-check` to do: that is, it enables Photo's built in duplicate check. Without this, duplicates are imported even if they already exist in the library. I think the option you really want is `--resume`.

```
-R, --resume                      Resume previous import. Note: data on each
                                  imported file is kept in a database in 
                                  '~/.local/share/osxphotos/osxphotos_imp
                                  ort.db'. This data can be used to resume a
                                  previous import if there was an error or the
                                  import was cancelled.
```

This is intended for your use case where photos are being added to a folder.  Photos that were previously successfully imported will be skipped when `--resume` is used.

**Comment by cbgrey** [comment]
Ohhhhh! That makes sense. Sorry for my misunderstanding. I will try "--resume" and turn off "--dup-check" to avoid the pop up. Appreciate the help!
